pkgs/tools/text/chars/default.nix

```
@@ -19,6 +20,10 @@ rustPlatform.buildRustPackage rec {
 
   buildInputs = lib.optionals stdenv.isDarwin [ Security ];
 
+  passthru = {
+    updateScript = gitUpdater { rev-prefix = "v"; };
```

The gitUpdater can not update `cargoHash`. Use the following instead:

```suggestion
    updateScript = nix-update-script { extraArgs = [ "--version=branch" ]; };
```